spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -149,6 +149,36 @@ to `WARNING` in `logback.xml`:
 
 ====
 
+[[configuring-rdbms]]
+== RDBMS configuration
```

Comment:
I think the configuration would be different for different server implementations, CF adds it's own configuration and so does k8s, so switching DB would not be the same instructions across the board